modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/menu.js

```diff
@@ -339,17 +339,13 @@ AUI.add(
 
 					var overlay = instance._overlay;
 
-					var align = overlay.get('align');
-
 					var listNode = menu.one('ul');
 
 					var listNodeHeight = listNode.get('offsetHeight');
 					var listNodeWidth = listNode.get('offsetWidth');
 
 					var modalMask = false;
 
-					align.points = instance._getAlignPoints(cssClass);
```

Comment:
Can we remove the `_getAlignPoints` method since this is removed now?

Reply:
Hello @jonmak08,

After testing, it does seem okay to remove `_getAlignPoints`.

Reply:
Got it.  Ok I will remove it.